src/AnalysisPrograms/Recognizers/Birds/NinoxStrenua.cs

```diff
@@ -0,0 +1,144 @@
+            AudioRecording audioRecording,
+            Config config,
+            TimeSpan segmentStartOffset,
+            Lazy<IndexCalculateResult[]> getSpectralIndexes,
+            DirectoryInfo outputDirectory,
+            int? imageWidth)
+        {
+            //class NinoxStrenuaConfig is defined at bottom of this file.
+            var genericConfig = (NinoxStrenuaConfig)config;
+            var recognizer = new GenericRecognizer();
+
+            // Use the generic recognizers to find all generic events.
+            RecognizerResults combinedResults = recognizer.Recognize(
+                audioRecording,
+                genericConfig,
+                segmentStartOffset,
+                getSpectralIndexes,
+                outputDirectory,
+                imageWidth);
+
+            var count = combinedResults.NewEvents.Count;
+            PowerfulOwlLog.Debug($"Event count before post-processing = {count}.");
+            if (combinedResults.NewEvents.Count == 0)
+            {
+                return combinedResults;
+            }
+
+            // ################### POST-PROCESSING of EVENTS ###################
+            // Following two commented lines are different ways of casting lists.
+            //var newEvents = spectralEvents.Cast<EventCommon>().ToList();
+            //var spectralEvents = events.Select(x => (SpectralEvent)x).ToList();
+
+            //NOTE:
+            // The generic recognizer has already done the following post-processing of events prior to returning a combined list of events.
+            // Generic post processing step 1: Combine overlapping events.
+            // Generic post processing step 2: Combine possible syllable sequences and filter on excess syllable count.
+            // Generic post processing step 3: Remove events whose bandwidth is too small or large.
+            // Generic post processing step 4: Remove events that have excessive noise in their side-bands.
+
+            // Additional post-processing steps are put here.
+            // NOTE: THE POST-PROCESSING STEPS BETWEEN HERE AND OF THE END OF METHOD ARE THE ONLY STEPS THAT MAKE THIS A DIFFERENT RECOGNIZER FROM THE GENERIC.
+            // TYPICALLY YOU WOULD PROCESS THE INDIVIDUAL TRACKS IN EACH METHOD LOOKING FOR A SPECIFIC TRACK SHAPE.
+
+            // POST PROCESSING METHODS HERE >>>>>>>>>>>>>>>>>>>>>>>>>>
+
+            PowerfulOwlLog.Info($"Final event count = {combinedResults.NewEvents.Count}.");
+            if (combinedResults.NewEvents.Count > 0)
+            {
+                int counter = 0;
+                foreach (var ev in combinedResults.NewEvents)
+                {
+                    counter++;
+                    var spEvent = (SpectralEvent)ev;
+                    PowerfulOwlLog.Info($"  Event[{counter}]: Start={spEvent.EventStartSeconds:f1}; Duration={spEvent.EventDurationSeconds:f2}; Bandwidth={spEvent.BandWidthHertz} Hz");
+                }
+            }
```

Comment:
This seems like a fair generic method. Maybe move it somewhere more appropriate?

Also: definitely should be a `log.Debug` - this kind of verbose output doesn't belong in Info